### PR TITLE
fix: padding on the settings page

### PR DESCRIPTION
**Pull request by themegaphoenix**
Add padding in the settings page.
Before:
image
After:
image

### CODE REVIEW

File: pages/settings/index.vue [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <template>
   <v-container>
-    <v-row>
-      <v-col cols="12" offset-lg="1" md="5" lg="4" class="pt-0 pb-4">
+    <v-row class="pt-3">
+      <v-col cols="12" offset-lg="1" md="5" lg="4" class="pt-3 pb-4">
```
Comment (ThibaultNocchi) [inline pages/settings/index.vue:4]: Can't we put the v-col top padding to 4? I'm just nitpicking though :p

Comment (themegaphoenix) [reply pages/settings/index.vue]: Done. 4 is even :)

File: pages/settings/index.vue [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <template>
   <v-container>
-    <v-row>
-      <v-col cols="12" offset-lg="1" md="5" lg="4" class="pt-0 pb-4">
+    <v-row class="pt-4">
+      <v-col cols="12" offset-lg="1" md="5" lg="4" class="pt-4 pb-4">
```
Comment (ThibaultNocchi) [inline pages/settings/index.vue:4]:
```suggestion
      <v-col cols="12" offset-lg="1" md="5" lg="4" class="py-4">
```

**Review by ThibaultNocchi — COMMENTED** [review]
In that case we can refactor even more ;p

File: pages/settings/index.vue (hunk 2 above)
Comment (themegaphoenix) [reply pages/settings/index.vue]: Thanks. I did not that existed :) I learned a new thing today

**Comment by ThibaultNocchi** [comment]
Nice! Can you squash your three commits into one please? :)

**Comment by themegaphoenix** [comment]
> Nice! Can you squash your three commits into one please? :)

done

**Review by ThibaultNocchi — APPROVED** [review]

**Comment by MrTimscampi** [comment]
@themegaphoenix Could you rebase the branch to get the reworked CI, please?

**Review by MrTimscampi — APPROVED** [review]

**Pull request merged**